src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -447,6 +452,8 @@ impl<'a> TaskContext<'a> {
                             if count % 10 == 0 {
                                 self.save_and_sync_coverage().await?;
                             }
+                            info!("report coverage");
```

Comment:
don't emit this event here. At this point the coverage is is not really reported yet because we have no update the files.